Chinese/脑经急转弯接龙/answers/an_polar_bear.md

```diff
@@ -0,0 +1 @@
+北极熊在北极，企鹅在南极。[下一个](../questions/tree.md)
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.